src/service/tatum/tatum.ts

```diff
@@ -84,11 +84,21 @@ export class TatumSDK {
 
     const mergedConfig = Utils.deepMerge(defaultConfig, config) as TatumConfig
 
+    LoggerUtils.setLoggerForEnv(mergedConfig, EnvUtils.isDevelopment(), EnvUtils.isBrowser())
+
     // TODO: check when rpc is customized if there is allowedBlocksBehind if not throw error or set default
     // TODO: Check if rpc works for other chains and all configurations are set correctly
 
     const id = TatumSDK.generateRandomString()
     Container.of(id).set(CONFIG, mergedConfig)
+    Container.of(id).set(LOGGER, mergedConfig.logger)
+
+    if (!mergedConfig.apiKey?.v3 || !mergedConfig.apiKey?.v4) {
```

Comment:
IMHO this is wrong. 
Both api keys are not required. When doing init it can have v3 or v4.

Reply:
This requires either v3 or v4

Reply:
Good catch